README.md

```diff
@@ -1,10 +1,12 @@
 # ExaCA temperature example files
-This repository contains supplementary temperature data files (the .txt files in the "Temperatures" subdirectory) for use with the ExaCA code (see https://github.com/LLNL/ExaCA). Within the ExaCA repository, there are 3 example files that use the external temperature data within this ExaCA-Data repository: examples/Inp_SimpleRaster.txt, which uses the file Temperatures/Master-Raster-150um.txt (a single layer of a raster pattern), examples/Inp_AMBenchMultilayer.txt, which uses the files Temperatures/1CA-FOAM-fluidflow.txt and Temperatures/2CA-FOAM-fluidflow.txt (a four layer Additive Manufacturing-like problem that alternates an odd and an even layer scan pattern), and examples/Inp_TwoLineTwoLayer.txt, which uses the file Temperatures/TwoLine.txt (segments from two overlapping line scan melt pools, repeated for two layers of a build).
+This repository contains supplementary temperature data files (the .txt files in the "Temperatures" subdirectory) for use with the ExaCA code (see https://github.com/LLNL/ExaCA). Within the ExaCA repository, there are 2 example files that use the external temperature data within this ExaCA-Data repository: examples/Inp_SingleLine.json, which uses the file Temperatures/Line.txt (a single line from a laser scan over a substrate), and examples/Inp_TwoLineTwoLayer.json, which uses the file Temperatures/TwoLine.txt (segments from two overlapping line scan melt pools, repeated for two layers of a build).
```

Comment:
I don't see `TwoLine`..

Reply:
You don't? I see it https://github.com/LLNL/ExaCA-Data/tree/main/Temperatures

Reply:
Oh my bad I don't know how I missed that